Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -271,10 +248,8 @@ public final class SecureEnclaveValet: NSObject {
 extension SecureEnclaveValet {
     
     // MARK: Public Class Methods
-
-    /// - Parameters:
-    ///   - identifier: A non-empty string that uniquely identifies a SecureEnclaveValet.
-    ///   - accessControl: The desired access control for the SecureEnclaveValet.
```

Comment:
Why'd you remove `accessControl` from the header doc?

Reply:
woof good catch. Merge conflict bit me here.